<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>userEditDialog</class>
 <widget class="QDialog" name="userEditDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>256</width>
    <height>222</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>170</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>отмена</string>
   </property>
  </widget>
  <widget class="QPushButton" name="saveButton">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>170</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>сохранить</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>56</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>имя</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>56</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>пароль</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="usernameEdit">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>60</y>
     <width>113</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="passwdEdit">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>90</y>
     <width>113</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="typeBox">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>120</y>
     <width>79</width>
     <height>23</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Aдмин</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Коммендант</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Студент</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>120</y>
     <width>56</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>тип</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>saveButton</sender>
   <signal>clicked()</signal>
   <receiver>userEditDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>169</x>
     <y>181</y>
    </hint>
    <hint type="destinationlabel">
     <x>127</x>
     <y>110</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>userEditDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>79</x>
     <y>181</y>
    </hint>
    <hint type="destinationlabel">
     <x>127</x>
     <y>110</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
